eng/pipelines/templates/jobs/mgmt-release.yml

```diff
@@ -39,21 +39,13 @@ jobs:
 
       - ${{ if and(eq(parameters.ShouldPublishToNuget, 'true'), eq(variables['Build.Repository.Name'], 'Azure/azure-sdk-for-net')) }}:
 
-        - task: Powershell@2
-          displayName: 'Verify Package Tags and Create Git Releases'
-          inputs:
-            filePath: ${{ parameters.BuildToolsPath }}/scripts/create-tags-and-git-release.ps1
-            arguments: >
-              -artifactLocation '$(PIPELINE.WORKSPACE)/packages'
-              -workingDirectory $(System.DefaultWorkingDirectory)
-              -packageRepository 'Nuget'
-              -releaseSha '$(Build.SourceVersion)'
-              -repoOwner 'Azure'
-              -repoName 'azure-sdk-for-net'
-            pwsh: true
-          env:
-            GH_TOKEN: $(azuresdk-github-pat)
-          condition: and(succeeded(), ne(variables['SkipCreateTagAndGitRelease'], 'true'))
+        - template: /eng/common/pipelines/templates/steps/create-tags-and-git-release.yml
+          parameters:
+            ArtifactLocation: '$(PIPELINE.WORKSPACE)/packages'
+            WorkingDirectory: $(System.DefaultWorkingDirectory)
+            PackageRepository: Nuget
+            ReleaseSha: $(Build.SourceVersion)
+            RepoId: Azure/azure-sdk-for-net
```

Comment:
You shouldn't need to set RepoId. [create-tags-and-git-release.yml](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/common/pipelines/templates/steps/create-tags-and-git-release.yml#L6C3-L6C35) defaults this to $(Build.Repository.Name) which is exactly what you want. On the same note, you might want to consider removing the l[ine that sets this in the normal release yml](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/stages/archetype-net-release.yml#L108) as well.

Reply:
I removed this and the workingdirectory parameter from both files